portability-core/src/main/java/org/dataportabilityproject/serviceProviders/microsoft/calendar/MicrosoftCalendarService.java

```diff
@@ -138,10 +138,8 @@ private CalendarModelWrapper getCalendarEvents(
             "https://outlook.office.com/api/v2.0/me/calendars/%s/calendarview?startDateTime=%s&endDateTime=%s",
             calendarId, formatTime(begin), formatTime(end));
     System.out.println("calendar: " + calendarId);
-    System.out.println(
-        "eventsUrl: "
-            + eventsUrl); // TODO: Determine why this URL works in the MS Oauth Playground but not
-                          // here
+    // TODO: Determine why this URL works in the MS Oauth Playground but not here
+    System.out.println("eventsUrl: " + eventsUrl);
```

Comment:
TODO remove these system outs

Reply:
Ack I'll do that in a folllowup PR.

Reply:
I think we can remove that code now that the Graph API-based importers/exporters are in the MS extension.